great_expectations_cloud/agent/actions/list_table_names.py

```diff
@@ -28,7 +28,7 @@ class ListTableNamesAction(AgentAction[ListTableNamesEvent]):
     @override
     def run(self, event: ListTableNamesEvent, id: str) -> ActionResult:
         datasource_name: str = event.datasource_name
-        datasource = self._context.get_datasource(datasource_name=datasource_name)
+        datasource = self._context.get_datasource(datasource_name)
```

Comment:
this should be updated to V1 syntax:
```suggestion
        datasource = self._context.data_sources.get(name=datasource_name)
```